sweep/sweeper.go

```diff
@@ -318,6 +340,37 @@ func (s *UtxoSweeper) collector(blockEpochs <-chan *chainntnfs.BlockEpoch,
 			}
 			s.pendingInputs[outpoint] = pendInput
 
+			// If the caller didn't specify a fee preference, we'll
+			// use the sweeper's default.
+			if input.feePreference == nil {
+				pendInput.feePreference = FeePreference{
+					ConfTarget: s.cfg.SweepTxConfTarget,
+				}
+			} else {
+				// Otherwise, we'll validate it to ensure the
+				// caller provided a sane value.
+				_, err := DetermineFeePerKw(
```

Comment:
This can be moved into `SweepInput`, before we hand off to this goroutine.

Reply:
This validation is now handled by the `bucketManager`.